airbyte-integrations/connectors/destination-pinecone/unit_tests/destination_test.py

```diff
@@ -66,6 +66,26 @@ def test_check_with_errors(self, MockedEmbedder, MockedPineconeIndexer):
         mock_embedder.check.assert_called_once()
         mock_indexer.check.assert_called_once()
 
+    def test_check_with_config_errors(self):
```

Comment:
unrelated to attribution change - these two tests are for improving test coverage.